update.xml

```diff
@@ -753,7 +766,7 @@
 
         <z2_plus>
             <maintainer>Yashodhan Sawardekar</maintainer>
-            <devicename>ZUK Z2 Plus</devicename>
+￼             <devicename>ZUK Z2 Plus</devicename>
```

Comment:
you messed around with this 
delete it

Reply:
sry next time i will use my pc :) to do all this done changes